src/modules/job-exec/job-exec.c

```diff
@@ -136,6 +136,7 @@ void jobinfo_decref (struct jobinfo *job)
         idset_destroy (job->critical_ranks);
```

Comment:
In commit message for "terminate job shell with SIGKILL after timeout",
s/rpocess/process/

Reply:
Fixed that and forced a push. Will set MWP.